### PR TITLE
Link to ActionView::Helpers top level namespace for all helpers

### PR DESCRIPTION
**Pull request by zzak**
Follow up to #48087 

After thinking about it, makes sense to link to the top level module so the reader can see all of the helpers available.

/cc @mikepmunroe since this was originally your patch, it could have been taken care of in the previous review; I've added you as a co-author. :bow:

### CODE REVIEW

File: guides/source/form_helpers.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ After reading this guide, you will know:
 
 --------------------------------------------------------------------------------
 
-NOTE: This guide is not intended to be a complete documentation of available form helpers and their arguments. Please visit [the Rails API documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html) for a complete reference.
+NOTE: This guide is not intended to be a complete documentation of available form helpers and their arguments. Please visit [the Rails API documentation](https://api.rubyonrails.org/classes/ActionView/Helpers.html) for a complete reference of all available helpers.
```
[the remark this answers is not in the document]
Comment (p8) [reply guides/source/form_helpers.md:20]: Good point. I've created an alternative solution:
https://github.com/rails/sdoc/pull/224

[the remark this answers is not in the document]
Comment (zzak) [reply guides/source/form_helpers.md:20]: Thanks both for the review and good call out! :bow:

[the remark this answers is not in the document]
Comment (zzak) [reply guides/source/form_helpers.md:20]: @p8 I think I prefer the first option, a single link to the AV helpers docs. Would be great to add some more info there too, and link back to the guide, etc. :pray:

[the remark this answers is not in the document]
Comment (mikepmunroe) [reply guides/source/form_helpers.md:20]: +1 on the first option

[the remark this answers is not in the document]
Comment (zzak) [reply guides/source/form_helpers.md:20]: @mikepmunroe Would you mind submitting a patch to update the link then? Bonus points for any additional info on that page even if it's just a link back to the guide, and a enumeration of the available helpers :bow:

**Pull request merged**